src/main/java/uk/gov/pay/directdebit/notifications/services/UserNotificationService.java

```diff
@@ -0,0 +1,104 @@
+        readMandateFailedEmailConfig(directDebitConfig.getNotifyConfig());
+        if (emailNotifyGloballyEnabled) {
+            int numberOfThreads = directDebitConfig.getExecutorServiceConfig().getThreadsPerCpu() * getRuntime().availableProcessors();
+            executorService = Executors.newFixedThreadPool(numberOfThreads);
+        }
+        this.directDebitConfig = directDebitConfig;
+        this.notificationClient = notificationClient;
+        this.metricRegistry = metricRegistry;
+        this.gatewayAccountService = gatewayAccountService;
+    }
+
+    public Future<Optional<String>> sendMandateFailedEmail(Payer payer, Transaction transaction) {
+        if (emailNotifyGloballyEnabled) {
+            Stopwatch responseTimeStopwatch = Stopwatch.createStarted();
+            return executorService.submit(() -> {
+                try {
+                    SendEmailResponse response = notificationClient
+                            .sendEmail(
+                                    mandateFailedTemplateId,
+                                    payer.getEmail(),
+                                    buildMandateFailedPersonalisation(transaction),
+                                    null);
+                    return Optional.of(response.getNotificationId().toString());
+                } catch (NotificationClientException e) {
+                    LOGGER.error("Failed to send mandate failed email, payment request id: {}, error {}", transaction.getPaymentRequestExternalId(), e);
+                    metricRegistry.counter("notify-operations.failures").inc();
+                    return Optional.empty();
+                } finally {
+                    responseTimeStopwatch.stop();
+                    metricRegistry.histogram("notify-operations.response_time").update(responseTimeStopwatch.elapsed(TimeUnit.MILLISECONDS));
+                }
+            });
+        }
+        return CompletableFuture.completedFuture(Optional.empty());
+    }
+
+    private HashMap<String, String> buildMandateFailedPersonalisation(Transaction transaction) {
+        GatewayAccount gatewayAccount = gatewayAccountService.getGatewayAccountFor(transaction);
+
+        HashMap<String, String> map = new HashMap<>();
+        map.put("org name", gatewayAccount.getServiceName());
+        map.put("org phone", "000-CAKE-000");
+        map.put("dd guarantee link", directDebitConfig.getLinks().getFrontendUrl() + "/direct-debit-guarantee");
+
+        return map;
+    }
+
+    private void readMandateFailedEmailConfig(NotifyClientFactory notifyClientFactory) {
+        emailNotifyGloballyEnabled = notifyClientFactory.isEmailNotifyEnabled();
+        mandateFailedTemplateId = notifyClientFactory.getMandateFailedTemplateId();
+
+        if (!emailNotifyGloballyEnabled) {
+            LOGGER.warn("Email notifications are globally disabled by configuration");
+        }
+        if (emailNotifyGloballyEnabled && isBlank(mandateFailedTemplateId)) {
```

Comment:
Force the check in the configuration itself rather than here.

Reply:
done 👍